src/activity_handlers.h

```diff
@@ -52,7 +52,7 @@ enum do_activity_reason : int {
     BLOCKING_TILE           // Something has made it's way onto the tile, so the activity cannot proceed
 };
 
-int butcher_time_to_cut( const player &u, const item &corpse_item, const butcher_type action );
```

Comment:
Why?

Reply:
hmm, no idea. thats weird. obvious mistake. will restore.

Reply:
ah thats due to clang-tidy aswell, about the const parameter.